Pod/Classes/GooglePlacesSearchController.swift

```diff
@@ -404,33 +404,28 @@ class GooglePlacesRequestHelpers {
     :param: parameters Dictionary of query string parameters
     :returns: The properly escaped query string
     */
-    private class func query(parameters: [String: AnyObject]) -> String {
+    fileprivate class func query(_ parameters: [String: AnyObject]) -> String {
         var components: [(String, String)] = []
-        for key in Array(parameters.keys).sort() {
+        for key in Array(parameters.keys).sorted() {
             let value: AnyObject! = parameters[key]
             components += [(key, "\(value)")]
         }
         
-        return components.map{"\($0)=\($1)"}.joinWithSeparator("&")
+        return components.map{"\($0)=\($1)"}.joined(separator: "&")
     }
     
-    private class func doRequest(urlString: String, params: [String: String], success: NSDictionary -> ()) {
-        if let url = NSURL(string: "\(urlString)?\(query(params))"){
-            
-            let request = NSMutableURLRequest(
-                URL:url
-            )
+    fileprivate class func doRequest(_ urlString: String, params: [String: String], success: @escaping (NSDictionary) -> ()) {
+			if let url = URL(string: "\(urlString)?\(query(params as [String : AnyObject]))"){
```

Comment:
Left line spacing through all PR is too huge. This is especially noticeable in this method.